### PR TITLE
Add docker

### PR DESCRIPTION
**Pull request by EduarfF**

### CODE REVIEW

**Review by HAIOVYI — APPROVED** [review]
GJ

File: .gitignore [hunk 1]
```diff
@@ -31,3 +31,4 @@ build/
 
 ### VS Code ###
 .vscode/
+.env
```
Comment (andrii-hoienko) [inline .gitignore]:
```suggestion
.env

```

**Review by andrii-hoienko — CHANGES_REQUESTED** [review]
Please check PR before pushing to review, you should make self review first, and then send to mentor`s review

**Review by andrii-hoienko — APPROVED** [review]
Nice!

**Pull request merged**